lib/core/rancher/step.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::core.rancher::help() {
+  echo 'Install Rancher Desktop as an alternative to Docker Desktop'
+}
+
+# Check if the step has been installed and return 0 if it has.
+# Otherwise return 1.
+function ih::setup::core.rancher::test() {
+
+    if command -v rdctl; then
+        echo "Rancher Desktop has been installed successfully"
+        return 0
+    else
+        echo "Install Rancher Desktop failed. Please contact platform support
```

Comment:
This message is not correct. The test is executed before we even attempt to install rancher. No message is needed.